frontend/app/views/support/ABTest.scala

```diff
@@ -52,9 +52,19 @@ object MessageCopyTest extends TestTrait {
     Variant("Copy - power", "power", 0, views.html.fragments.giraffe.contributeMessage("The powerful won't investigate themselves. That's why we need you.")),
     Variant("Copy - mutual", "mutual", 0, views.html.fragments.giraffe.contributeMessage("Can't live without us? The feeling's mutual.")),
     Variant("Copy - everyone", "everyone", 0, views.html.fragments.giraffe.contributeMessage("If everyone who sees this chipped in the Guardian's future would be more secure.")),
+    Variant("Copy - everyone", "everyoneinline", 0, views.html.fragments.giraffe.contributeMessage("If everyone who sees this chipped in the Guardian's future would be more secure.")),
```

Comment:
I think there should be! but there isn't in the spreadsheet. The text here doesn't actualy change anything on our end currently.